### PR TITLE
Find outer element tag with inner text when element tag is not provided

### PR DESCRIPTION
**Pull request by vyshakhj**
This PR is linked to issue #28

### CODE REVIEW

File: tests/test_soup.py [hunk 1]
```diff
@@ -100,6 +100,13 @@ def fake_html_6():
     return html
 
 
+def test_find_inner_text():
+    html = """<p>&pound;600m</p>"""
+    soup = Soup(html)
+    result = soup.find("p").text
```
Comment (maxhumber) [inline tests/test_soup.py:106]: This is fine! But the test case that isn't working is actually: 

```python
result = soup.text
```

Comment (vyshakhj) [reply tests/test_soup.py]: Ok, so we need to get the outer tag and then get the contents inside the tag.

File: gazpacho/soup.py [hunk 2]
```diff
@@ -56,11 +56,17 @@ def __init__(self, html: Optional[str] = None) -> None:
...
         self.tag: str = ""
         self.attrs: Optional[Dict[str, Any]] = None
-        self.text: str = ""
+        self.text: str = self.inner_text()
 
     def __repr__(self) -> str:
         return self.html
 
+    def inner_text(self):
+        element = re.match('<(.+)>.*>', self._html)
```
Comment (maxhumber) [inline gazpacho/soup.py:65]: Nice! 🥫 this looks great

**Review by maxhumber — APPROVED** [review]

File: gazpacho/soup.py (hunk 2 above)
Comment (vyshakhj) [reply gazpacho/soup.py:65]: Thanks, happy to contribute more.

**Pull request merged**